Xamarin.Forms.Build.Tasks/SetPropertiesVisitor.cs

```diff
@@ -17,6 +17,75 @@ namespace Xamarin.Forms.Build.Tasks
 {
 	class SetPropertiesVisitor : IXamlNodeVisitor
 	{
+		class Borrower
+		{
+			readonly ILContext context;
+			readonly INode node;
+
+			readonly Dictionary<string, (IEnumerable<Instruction>, FieldDefinition)> dictionary =
+				new Dictionary<string, (IEnumerable<Instruction>, FieldDefinition)>();
+
+			public Borrower(ILContext context, INode node)
+			{
+				this.context = context;
```

Comment:
no .this and on the line below. Other than that, looks good to me

Reply:
The field and arguments will be confused without `this`. How should I deal with that?

Reply:
Ah OK, I missed that, then it's all good

Reply:
The private fields names should be prefixed with an underscore as mentioned in the [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)